puppet/puppet.py

```diff
@@ -4,10 +4,16 @@
 class Puppet(object):
     """Python wrapper for Puppet REST API"""
 
-    def __init__(self, host='localhost', port=8140, key_file=None,
-        cert_file=None):
+    def __init__(self, host='localhost', port=8140, parser='yaml',
```

Comment:
Move parser argument after cert_file to retain backwards-compatibility for positional arguments? Also update order in `README.md`.